pypeit/core/wavecal/autoid.py

```diff
@@ -1121,6 +853,215 @@ def full_template(spec, lamps, par, ok_mask, det, binspectral, nsnippet=2,
     # Finish
     return wvcalib
 
+def echelle_wvcalib(spec, orders, spec_arxiv, wave_arxiv, lamps, par, ok_mask=None, use_unknowns=True, debug_all=False,
+                    debug_peaks=False, debug_xcorr=False, debug_reid=False, debug_fits=False, nonlinear_counts=1e10):
+    r"""
+    Algorithm to wavelength calibrate echelle data based on a predicted or archived wavelength solution
+
+    Args:
+    spec :  float ndarray shape of (nspec, norders)
```

Comment:
Please make this a valid Google or numpy style

Reply:
Same applies for the full_template method on line 662 of the same module, which is actually in the same format as my doc string. I suggest we leave them both as is, but you are welcome to submit a PR to tidy these up everywhere if you like.